core/services/ocr2/plugins/ocr2keeper/evm21/upkeep_state_store_test.go

```diff
@@ -1,72 +1,48 @@
 package evm
 
 import (
-	"fmt"
 	"math/big"
 	"testing"
 
 	ocr2keepers "github.com/smartcontractkit/ocr2keepers/pkg"
 	"github.com/stretchr/testify/require"
 
 	"github.com/smartcontractkit/chainlink/v2/core/logger"
+	"github.com/smartcontractkit/chainlink/v2/core/services/ocr2/plugins/ocr2keeper/evm21/core"
 )
 
 var (
 	upkeepId1 = big.NewInt(100)
 	upkeepId2 = big.NewInt(200)
 	trigger1  = ocr2keepers.Trigger{
-		BlockNumber: 95,
+		BlockNumber: block1,
 		BlockHash:   "0x1231eqwe12eqwd",
+		Extension: core.LogTriggerExtension{
+			LogIndex: 1,
+			TxHash:   "0x1234567890123456789012345678901234567890123456789012345678901234",
+		},
 	}
 	trigger2 = ocr2keepers.Trigger{
-		BlockNumber: 125,
+		BlockNumber: block3,
 		BlockHash:   "0x1231eqwe12eqwd",
+		Extension: core.LogTriggerExtension{
+			LogIndex: 1,
+			TxHash:   "0x1234567890123456789012345678901234567890123456789012345678901234",
+		},
 	}
-	payload2 = ocr2keepers.NewUpkeepPayload(upkeepId1, conditionalType, blockKey2, trigger1, []byte{})
-	payload3 = ocr2keepers.NewUpkeepPayload(upkeepId2, logTriggerType, blockKey2, trigger1, []byte{})
-	payload4 = ocr2keepers.NewUpkeepPayload(upkeepId1, logTriggerType, blockKey1, trigger2, []byte{})
-	payload5 = ocr2keepers.NewUpkeepPayload(upkeepId1, logTriggerType, blockKey3, trigger1, []byte{})
+	payload2, _ = core.NewUpkeepPayload(upkeepId1, conditionalType, trigger1, []byte{})
```

Comment:
check err

Reply:
These are generated outside of tests as variables, I'm not sure how to check error?